eng/common/scripts/FilterPoliCheckResults.ps1

```diff
@@ -3,38 +3,45 @@
 Filters PoliCheck Result.
 .DESCRIPTION
 This script will read data speciefied in one or more PoliCheckAllowList.yml files,
-It then reamoves all allwed entries from the PoliCheckResult 
+It then removes all allowed entries from the PoliCheckResult 
 .PARAMETER PoliCheckResultFilePath
 The Path to the PoliCheck Result. Usually named PoliCheck.sarif
-.PARAMETER ServiceDirtectory
+.PARAMETER ServiceDirectory
 If the PoliCheck scan is scoped to a particular service provide the ServiceDirectory
+.PARAMETER AllowListLocation
+A path to a folder containing yml defined policheck allowlist
 .EXAMPLE
-PS> ./FilterPoliCheckResults.ps1 -PoliCheckResultFilePath .\PoliCheck.sarif
+PS> ./FilterPoliCheckResults.ps1 -PoliCheckResultFilePath .\PoliCheck.sarif -ServiceDirectory <servicedirname> -AllowListLocation <location>
 #>
 [CmdletBinding()]
 param(
   [Parameter(Mandatory=$true)]
   [String] $PoliCheckResultFilePath,
-  [String] $ServiceDirtectory
+  [Parameter(Mandatory=$true)]
+  [String] $ServiceDirectory,
+  [Parameter(Mandatory=$true)]
+  [String] $AllowListLocation
 )
 
 . "${PSScriptRoot}\logging.ps1"
```

Comment:
Any reason we should not import common.ps1?

Reply:
Nope, its just that only the logging is being used at the moment.